en_partut-ud-train.conllu

```diff
@@ -43252,8 +43252,8 @@
 26	supposes	suppose	VERB	V	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	6	acl	_	_
 27	he	he	PRON	PE	Gender=Masc|Number=Sing|Person=3|PronType=Prs	31	nsubj	_	_
 28	is	be	AUX	V	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	31	cop	_	_
-29	as	as	ADP	E	_	31	case	_	_
-30	well	well	ADV	B	_	29	fixed	_	_
+29	as	as	ADP	E	_	30	case	_	_
```

Comment:
Should be an ADV/advmod I think

Reply:
Sounds good.  So you are good with this phrase not being `fixed`?  It looks more like a comparison to me, so I parsed it that way.